flask_resty/testing.py

```diff
@@ -28,11 +29,11 @@ def get_meta(response):
 
 def assert_value(actual, expected):
     if isinstance(expected, dict):
-        assert isinstance(actual, dict)
+        assert isinstance(actual, Mapping)
         for k, v in expected.items():
             assert_value(actual.get(k, None), v)
-    elif isinstance(expected, list):
-        assert isinstance(actual, list)
+    elif isinstance(expected, Sequence) and not isinstance(expected, String):
```

Comment:
this does not match the above where you check for `dict` on `expected`

why do we need to assert on the instance of `actual`, anyway? `[] == {}` is false, right?